### ISSUE TITLE
KeyNotFoundException on API service

### ISSUE THREAD

**Issue by caribgit** [NCRCounterpointAPI/APIGuide#100]
We noticed we could not authenticate on the API service. We were pretty sure our credentials were correct. After several tries were unsuccessful, we decided to restart the API service on the server. This resolved the issue, and the same password worked right away. We checked the memory usage before restarting at it was about 74 MB so nothing crazy.

Has anyone else experienced this? Is it a known issue in certain circumstances?

**Comment by tommymh** [comment]
Yes I am actively experiencing this issue, restarting the API service does not appear to work currently though.
